### DISCUSSION TITLE
Welcome to kcp!

### DISCUSSION THREAD

**Discussion by smarterclayton** [kcp-dev/kcp#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clarkmcc** [comment]
Hello all! I thought I'd introduce myself here because this project caught my eye. I've been a Kubernetes user for about two years now and have been inside the codebase for about a year. A little over a year ago my software company decided to use the https://github.com/kubernetes-sigs/apiserver-builder-alpha project as the core of a small side product that some of our customers were looking for. As we dug into the fundamental Kubernetes principles, we really began to see (like you all) the potential of concepts like the controller-resource pattern for use-cases outside of just container orchestration.

We are getting close to our first release of our product and I have a soft spot for these projects that are attempting to make Kubernetes more than just container-orchestration.

I'm excited to see this project grow and hope that I can be of use to the project if needed.
